### ISSUE TITLE
CM disc = 0?

### ISSUE THREAD

**Issue by sehlen** [LMFDB/lmfdb#731]
On /ModularForm/GL2/Q/holomorphic/3/6/1/a/ the CM field is displayed as Q(\sqrt{0}), which does not make sense.

**Comment by JohnCremona** [comment]
In my refactor branch it is fine.  We'll check again after merging.

**Comment by JohnCremona** [comment]
It is fine with fredrik's refactor branch too so I will close this once that is merged into mine.
